tests/plotting/test_theme.py

```diff
@@ -492,16 +493,38 @@ def test_theme_eq():
 
 
 def test_plotter_set_theme():
-    # test that the plotter theme is set to the new theme
+    """Test that the plotter theme is set to the new theme"""
+
     my_theme = pv.plotting.themes.Theme()
     my_theme.color = [1.0, 0.0, 0.0]
     pl = pv.Plotter(theme=my_theme)
     assert pl.theme.color == my_theme.color
     assert pv.global_theme.color != pl.theme.color
 
+
+def test_plotter_set_theme_to_instance():
```

Comment:
Naming question/suggestion for this test.  This is really about testing the attribute setter than the fact that it is an instance?

Reply:
Feel free to propose another test name 😄

Reply:
`test_plotter_theme_attribute_setter`?